### PR TITLE
Decouple VS Code Screenshots, Update Windows Versions

### PR DESCRIPTION
**Pull request by jamesjuett**
This PR:

**Decouples `setup_vscode_wsl.md` and `setup_vscode_macos.md`**.
They no longer share any images except for the tiny "icon" images used in the debugging section.

**Renames image files** based on the version they're used with.
If an image was previously shared, two files are created. For example:
- `vscode030.png` was previously shared. It now becomes two files, `vscode_macos_030.png` and `vscode_wsl_030.png`.
- `vscode037.png` was only used for the Mac tutorial. It is renamed to `vscode_macos_037.png`.

**Updates Windows Screenshots**
Screenshots that were previously taken on a Mac but used in the Windows tutorial have been updated. For example, `vscode_wsl_030.png` is a new screenshot. The best way to see the actual changes to images is to look at the diff from 5f94d59.

**Suggested Validation**
- Global search or `grep` to ensure there are no lingering references to files with the old naming scheme. For example, a search for `vscode\d+.png` should yield no results, but a search for `vscode_wsl_\d+.png` or `vscode_macos_\d+.png` will.
- Ensure no files remain with names containing the old naming scheme. For example, `find docs/images -name "*vscode*.png" | grep 'vscode[0-9]'` should yield no results.
- Review the diff on 5f94d59 to verify the windows version conveys the same meaning as the previous mac version.
- Double check the site preview and ensure images in both tutorials look ok and there are no broken images.
- Review the diffs on the `.md` files to verify I changed the links correctly (this is probably not necessary, since I changed them in an automated way using multiple cursors).

### CODE REVIEW

File: docs/setup_vscode_macos.md [hunk 1]
```diff
@@ -475,7 +475,7 @@ First, you should already have the `C/C++` extension installed ([Instructions](#
 
 Next, open VS Code's Command Palette with `View > Command Palette` or with the keyboard shortcut `cmd + shift + P`.  Search for and select `C/C++: Edit Configurations (JSON)`.  This will open the file `c_cpp_properties.json`.
 
-<img src="images/vscode150.png" width="480px" />
+<img src="images/vscode_macos_150.png" width="480px" />
```
Comment (amirkamil) [inline docs/setup_vscode_macos.md:478]: I think line 505 below (the one that mentions WSL) is not relevant to macOS. Can you remove it?

File: docs/setup_vscode_wsl.md [hunk 2]
```diff
@@ -244,36 +244,36 @@ $ g++ -g main.cpp -o main.exe
...
 If you accidentally open VS Code from Windows mode, click on the green icon in the lower left hand corner and then select "Reopen Folder in WSL".
 
-<img src="images/vscode070.png" width="720">
+<img src="images/vscode_wsl_070.png" width="720">
 </div>
 
 Select the file you would like to run.  Navigate to the debugging pane.
 
-<img src="images/vscode030w.png" width="768px" />
+<img src="images/vscode_wsl_030w.png" width="768px" />
```
Comment (amirkamil) [inline docs/setup_vscode_wsl.md:256]: Should be `vscode_wsl_030.png`.

File: docs/setup_vscode_wsl.md [hunk 3]
```diff
@@ -244,36 +244,36 @@ $ g++ -g main.cpp -o main.exe
...
 
 Click "create a launch.json file".
 
-<img src="images/vscode031w.png" width="768px" />
+<img src="images/vscode_wsl_031w.png" width="768px" />
```
Comment (amirkamil) [inline docs/setup_vscode_wsl.md:260]: Should be `vscode_wsl_031.png`.

File: docs/setup_vscode_wsl.md [hunk 4]
```diff
@@ -244,36 +244,36 @@ $ g++ -g main.cpp -o main.exe
...
 
 If you are prompted to select a debugger, select C++ (GDB/LLDB).
 
-<img src="images/vscode031a.png" width="768px" />
+<img src="images/vscode_wsl_031a.png" width="768px" />
 
 Click "Add Configuration".  If the button does not appear in the bottom-right corner, select "Run" from the top menu, then select "Add Configuration".
 
-<img src="images/vscode032w.png" width="768px" />
+<img src="images/vscode_wsl_032w.png" width="768px" />
```
Comment (amirkamil) [inline docs/setup_vscode_wsl.md:268]: Should be `vscode_wsl_032.png`.

File: docs/setup_vscode_wsl.md (hunk 2 above)
Comment (amirkamil) [reply docs/setup_vscode_wsl.md]: Fixed in [edefb88](https://github.com/eecs280staff/tutorials/pull/154/commits/edefb88e37bde4d507cd59389403addf3e5d4856).

**Review by amirkamil — APPROVED** [review]
Made some fixes and tweaks, but otherwise LGTM.

**Comment by jamesjuett** [comment]
Thanks @amirkamil! Good catch on the "w" screenshots.

**Pull request merged**